int("six")

#This is called value error